modules/ROOT/pages/customize-kubernetes-crd.adoc

```diff
@@ -0,0 +1,179 @@
+== Customize Mule Application Kubernetes Resources
+
+Anypoint Runtime Fabric enables you to customize your Mule application Kubernetes resources through the custom resource `kubernetestemplates.rtf.mulesoft.com` template.
+
+Name the `kubernetestemplate` resource as `mule-application`. You can define the template in two different namespaces:
+
+* Applications namespace +
+The template is applied to all Mule applications deployed to the given namespace.
```

Comment:
```suggestion
The template is applied to all Mule apps deployed to the given namespace.
```
Use "app" except when "Application" is part of a license, permission, or product name